### DISCUSSION TITLE
Cannot access compute node after nfs mount

### DISCUSSION THREAD

**Discussion by TheFourC** [chpc-tech-eval/scc#141] [Q&A] [answered]
After the steps to create the shared home directory, access to the compute node is unavailable. We cannot proxy jump nor can we ssh from the head node to the compute node. The screenshot below contains the error messages received when trying to access the compute node. If we unmount the shared directory, access to the compute node is restored.  

![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/134003092/3e924e69-285d-474d-b7d0-bfa8496e3b87)

**Comment by nyameko** [comment, marked as answer]
Hi TheFourC,

I hope that you've managed to fix your error.

Please refer to the discussion about SELinux 

Good Luck!

**Comment by TheFourC** [comment]
Dear Sir, yes Thank you my team resolved the issue
